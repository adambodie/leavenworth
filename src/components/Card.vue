<template>
	<div>
		<div class="options">
			<div class="type">
			<select v-on:change="filterList">
				<option value="">Select type of cards...</option>
				<option v-for="(item, index) in uniqueCardsList" :key="index">{{item}}</option>
			</select>
			</div>
			<div class="filter">
				<select v-model="value" v-on:change="filter()">
					<option value="">Filter by Amount...</option>
					<option v-for="(card, index) in cards" v-bind:value="index" :key="index" >
						{{ index + 1 }}
					</option>
				</select>
			</div>
		</div>
		<transition-group name="flip" tag="ul" class="card-list">
			<li 
				v-bind:class="{ invisible: !card.visible }"
				v-show="(index <= value && type === '') || type === card.type" 
				v-for="(card, index) in shuffleCards" 
				:key="card.id"
			>
				<vue-flip :active-click="true" width="259px" height="345px" transition="0.4s" >
					<p slot="front" class="card" key="english">{{card.english}}
						<span class="language">English</span>
						<span class="delete-card" v-on:click="showCards(card)">X</span>
					</p>
					<p slot="back" class="card" key="german">{{card.german}}
						<span class="language">German</span>
						<span class="delete-card" v-on:click="showCards(card)">X</span>
					</p>
				</vue-flip>
			</li>
		</transition-group>
	</div>
</template>

<script>
import VueFlip from 'vue-flip';
import { mapState, mapMutations } from 'vuex';

export default {
	components: {
		'vue-flip': VueFlip
	},
	methods: mapMutations({
		filterList: function(){
			this.$store.commit('filterList');
		},
		filter: function(){
			this.$store.commit('filter');
		},
		showCards: function(card) {
			this.$store.commit('showCards', card);
		}
	}),
	computed: mapState({
		cards: state => state.cards,
		value: state => state.value,
		type: state => state.type,
		visible: state => state.visible,
		uniqueCardsList: function() {
			const types = [];
			this.cards.forEach((card) => {
				if(!types.includes(card.type)) {
					types.push(card.type);
				}
			});
			return types;
		},
		shuffleCards: function() {
			const array = this.cards;
			let currentIndex = array.length, temporaryValue, randomIndex;
			while (0 !== currentIndex) {
				randomIndex = Math.floor(Math.random() * currentIndex);
				currentIndex -= 1;
				temporaryValue = array[currentIndex];
				array[currentIndex] = array[randomIndex];
				array[randomIndex] = temporaryValue;
			}
			return array;
		}
	})
}
</script>
