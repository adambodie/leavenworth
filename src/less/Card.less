//Imports

@import (css) url('https://fonts.googleapis.com/css?family=Shrikhand');

// Variables
@images: 'http://leavenworth.bodiewebdesign.com/images';
@font-family: 'Shrikhand', cursive;
@banner-color: #c79a61;

// Mixins
.card-image-size() {
	width: 225px;
	height: 300px;
}

.title-font-shadow() {
		font-family: @font-family;
		color: white;
		text-shadow: 2px 2px 4px black;
}

//Main Styles

body {
    font-family: 'Open Sans', sans-serif; 
    background-color: #a86957;
}

.border {
	position: absolute;
	left: 0;
	right: 0;
	width: 100%;
}

// Header Styles
#header {
	padding: 10px;
	background-color: @banner-color;
	h1 {
		font-size: 2.5em;
		font-family: @font-family;
		color: white;
		text-shadow: 4px 4px 8px darken(white, 80%);
	}
}

// Intro Styles
.intro-body {
	display: flex;
}

.paragraph {
	padding: 20px;
}

.map {
	img {
		padding: 20px;
	}
}

// Footer Styles
#footer {
	padding: 10px;
	background-color: @banner-color;
	font-family: @font-family;
	color: white;
}


// Jumbotron Styles
.jumbotron {
	margin: 30px;
	box-shadow: 4px 4px 8px #23201b;
	h1 {
		.title-font-shadow;
		text-align: center;
		font-size: 2em;
		padding: 20px;
		background-color: @banner-color;
		box-shadow: 4px 4px 8px #23201b;
	}
}

// Loading Background Styles
.loading-background {
    background: #a86957!important;
    opacity: 1!important;
}


// Photo Gallery Styles
.caption {
	position: absolute;
	left: 0;
	bottom: 10px;
	padding: 10px 10px;
	font-size: 14px;
	.title-font-shadow;
	text-align: center;
	width: 100%;
}

.owl-carousel .owl-item img {
    display: block;
    width: auto;
    height: 450px!important;
    margin: 0 auto;
}

.owl-item {
	img {
		opacity: 0.5;
	}
	.caption {
		display: none;
	}
}

.owl-nav {
	margin-top: 0;
}
.center {
	img {
		opacity: 1;
	}
	.caption {
		display: block;
	}
}

.owl-theme .owl-nav [class*=owl-] {
    font-size: 36px!important;
    margin: 20px;
    background: @banner-color!important;
    width: 50px;
    height: 50px;
    border-radius: 100%;
    box-shadow: 2px 2px 4px black;
    &:focus {
		outline: none;
    }
}

// Cards Styles
.options {
	display: flex;
	justify-content: space-around;
	align-items: center;
}
ul {
    padding-left: 0;
    display: flex;
    flex-flow: wrap;
    justify-content: center;
  }

li {
	list-style-type: none;
    padding: 10px 10px;
    transition: all 0.5s ease;
		&:nth-child(3n + 0) .card {
			background: url('@{images}/card1.JPG');
		}
		&:nth-child(3n + 1) .card {
			background: url('@{images}/card2.JPG');
		}
		&:nth-child(3n + 2) .card {
			background: url('@{images}/card3.JPG');
		}
	}


.card {
    display: flex;
    justify-content: center;
    align-items: center;
    .card-image-size;
    .title-font-shadow;
    border-radius: 20px;
    text-align: center;
    position: relative;
    font-size: 2em;
    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
    will-change: transform;
  }

.delete-card {
	position: absolute;
	right: 0;
	top: 0;
	padding: 10px 15px;
	opacity: 0.4;
	transition: all 0.5s ease;
	font-family: 'Open Sans', sans-serif;
	font-size: 14px;
	cursor: pointer;
	&:hover {
		color: black;
	}
}
.language {
	position: absolute;
	left: 0;
	top: 0;
	padding: 10px 15px;
	transition: all 0.5s ease;
	font-size: 14px;
}

.invisible {
	display: none;
}

// Transition Styles
.flip-enter-active {
  transition: all 0.4s ease;
}
  
.flip-leave-active {
  display: none;
}
  
.flip-enter, .flip-leave {
  transform: rotateY(180deg);
  opacity: 0;
}