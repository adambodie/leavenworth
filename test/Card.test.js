import Vue from 'vue'
import { createLocalVue } from '@vue/test-utils'
import Vuex from 'vuex'
import cardStore from '../src/store/index';
import Card from '../src/components/Card.vue';
import { shallowMount } from '@vue/test-utils';
import { cloneDeep } from 'lodash'



test('returns value from store', () => {
  const localVue = createLocalVue()
  localVue.use(Vuex)
  const store = new Vuex.Store(cloneDeep(cardStore));
  expect(store.state.value).toBe(9)
})

test('returns value from store', () => {
	const localVue = createLocalVue()
	localVue.use(Vuex)
	const store = new Vuex.Store(cloneDeep(cardStore));
	const wrapper = shallowMount(Card, { store, localVue })
	expect(wrapper.element).toMatchSnapshot()
})
