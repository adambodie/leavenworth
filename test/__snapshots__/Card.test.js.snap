// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns value from store 1`] = `
<div>
  <div
    class="options"
  >
    <div
      class="type"
    >
      <select>
        <option
          value=""
        >
          Select type of cards...
        </option>
         
        <option>
          Cuisine
        </option>
        <option>
          Geography
        </option>
        <option>
          Animal
        </option>
        <option>
          Transportation
        </option>
        <option>
          Person
        </option>
        <option>
          Greeting
        </option>
        <option>
          Plants
        </option>
        <option>
          Weather
        </option>
        <option>
          Place
        </option>
        <option>
          Tool
        </option>
      </select>
    </div>
     
    <div
      class="filter"
    >
      <select>
        <option
          value=""
        >
          Filter by Amount...
        </option>
         
        <option
          value="0"
        >
          
						1
					
        </option>
        <option
          value="1"
        >
          
						2
					
        </option>
        <option
          value="2"
        >
          
						3
					
        </option>
        <option
          value="3"
        >
          
						4
					
        </option>
        <option
          value="4"
        >
          
						5
					
        </option>
        <option
          value="5"
        >
          
						6
					
        </option>
        <option
          value="6"
        >
          
						7
					
        </option>
        <option
          value="7"
        >
          
						8
					
        </option>
        <option
          value="8"
        >
          
						9
					
        </option>
        <option
          value="9"
        >
          
						10
					
        </option>
        <option
          value="10"
        >
          
						11
					
        </option>
        <option
          value="11"
        >
          
						12
					
        </option>
        <option
          value="12"
        >
          
						13
					
        </option>
        <option
          value="13"
        >
          
						14
					
        </option>
        <option
          value="14"
        >
          
						15
					
        </option>
        <option
          value="15"
        >
          
						16
					
        </option>
        <option
          value="16"
        >
          
						17
					
        </option>
        <option
          value="17"
        >
          
						18
					
        </option>
        <option
          value="18"
        >
          
						19
					
        </option>
        <option
          value="19"
        >
          
						20
					
        </option>
        <option
          value="20"
        >
          
						21
					
        </option>
        <option
          value="21"
        >
          
						22
					
        </option>
        <option
          value="22"
        >
          
						23
					
        </option>
        <option
          value="23"
        >
          
						24
					
        </option>
        <option
          value="24"
        >
          
						25
					
        </option>
        <option
          value="25"
        >
          
						26
					
        </option>
        <option
          value="26"
        >
          
						27
					
        </option>
        <option
          value="27"
        >
          
						28
					
        </option>
      </select>
    </div>
  </div>
   
  <span
    class="card-list"
    name="flip"
    tag="ul"
  >
    <li
      class=""
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Clouds
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Wolken
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Tree
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Baum
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Flowers
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Blumen
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Pipe
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Rohr
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Bear
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          BÃ¤r
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Island
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Insel
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Goodbye
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Auf Wiedersehen
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Waterfall
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Wasserfall
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Library
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Bibliothek
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Blackbird
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Amsel
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Cheers
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Prost
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Sun
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Sonne
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Street
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          StraÃe
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Highway
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Autobahn
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          City Hall
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Rauthaus
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Knight
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Ritter
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Chocolate
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Shockolade
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Beer
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Bier
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Goat
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Ziege
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Nutcracker
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Nussknacker
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Waitress
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Kellnerin
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Sausage
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Wurst
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Mountain
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Berg
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          River
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Fluss
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Welcome
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Willkommen
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Bridge
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          BrÃ¼cke
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Icicle
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Eiszapfen
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
    <li
      class=""
      style="display: none;"
    >
      <vue-flip-stub
        activeclick="true"
        height="345px"
        transition="0.4s"
        width="259px"
      >
        <p
          class="card"
        >
          Rain
						
          <span
            class="language"
          >
            English
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
         
        <p
          class="card"
        >
          Regen
						
          <span
            class="language"
          >
            German
          </span>
           
          <span
            class="delete-card"
          >
            X
          </span>
        </p>
      </vue-flip-stub>
    </li>
  </span>
</div>
`;
